src/java/org/apache/cassandra/utils/JVMKiller.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.utils;
+
+/**
+ * An interface implemented to kill the JVM abnormally.
+ *
+ * It's currently implemented by {@link JVMStabilityInspector}
```

Comment:
`JVMStabilityInspector` contains class `Killer`, which implements the interface. Will it be better to be more specific?
E.g.:
```suggestion
 * It's currently implemented in {@link JVMStabilityInspector}
```
Or 
```suggestion
 * It's currently implemented by {@link JVMStabilityInspector.Killer}
```

Reply:
👍🏻  I like the second option